src/components/skybox.js

```diff
@@ -280,6 +288,20 @@ AFRAME.registerComponent("skybox", {
 
       this.el.object3D.matrixNeedsUpdate = true;
     }
+
+    this.updateEnvironmentMap();
```

Comment:
Hmm, update gets called on init, so it seems like you would end up rendering twice the first time. Perhaps we don't need the one in the requestAnimationFrame after all?

Reply:
For some reason it needs to be called the frame after the skybox is initialized so I do need the requestAnimationFrame callback. I'm not 100% certain why though.

Reply:
Huh, it needs to be called in init and in the requestAnimationFrame callback... I don't know why.